README.md

```diff
@@ -37,8 +37,12 @@ A curated list of awesome things and projects built by Algerian developpers.
   - [Rungen](https://github.com/youknowriad/rungen) - A generic runtime around async flow in javascript
 - PHP
   - [Phyber](https://github.com/ghousseyn/phiber) - Build light-weight lightning-fast PhP applications with Phiber
+  - [FrisAI](https://github.com/OGFris/FrisAI) - FrisAI is an A.I. based chat-bot made with PHP.
```

Comment:
Would you mind alphabetically ordering these?

Reply:
yea my bad